android/build.gradle.kts

```diff
@@ -30,8 +39,12 @@ android {
 dependencies {
 	api(project(":library"))
 
-	implementation(kotlin("stdlib-jdk7"))
+	implementation(kotlin("stdlib-jdk8"))
+
+	implementation("androidx.core:core-ktx:1.2.0")
+	implementation("androidx.annotation:annotation:1.1.0")
 
+	implementation("com.google.code.gson:gson:2.8.6")
```

Comment:
Why do we need `gson` here?

Reply:
We don't need it, it's a leftover from a previous commit. whoops